package.json

```diff
@@ -8,20 +8,21 @@
   "scripts": {
     "sass": "sass --style=compressed _sass/main.scss _site/css/main.css",
     "watch:eleventy": "eleventy --serve",
-    "watch:sass": "yarn sass -- --watch",
-    "serve": "run-p sass watch:*",
+    "watch:sass": "yarn sass --watch --",
+    "serve": "run-p sass 'watch:*'",
```

Comment:
Note to reviewers: this change was required to avoid the two following errors:

```console
$ yarn serve               
No matches found: "watch:*"
```

```console
$ yarn watch:sass 
Only two positional args may be passed.
```